src/farmer.rs

```diff
@@ -86,37 +87,45 @@ impl CacheDescription {
 }
 
 /// Description of the plot
```

Comment:
BTW, there is a data structure with details about the plot in farmer already and it has more details in it.

Reply:
Yeah, but this is just a description. The same thing as this:

https://github.com/subspace/subspace/blob/main/crates/subspace-farmer/src/bin/subspace-farmer/main.rs#L113-L119